### PR TITLE
Gtk3.0 port

### PR DESCRIPTION
**Pull request by rhoit**
using python gi.objects
glade generation for gtk+ version = 3.12
tested in python 3.4.3
using gi.**version** = 3.16.1

### CODE REVIEW

**Comment by jcollado** [comment]
@rhoit Thanks for your PR! When I wrote this code I was still using pygtk, but an update to Gtk3 is needed.

File: browser.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
+#!/usr/bin/env python
+
 import json
 import logging
 
-import gobject
-import webkit
+from gi.repository import WebKit
+from gi import _gobject as gobject
```
Comment (jcollado) [inline browser.py]: @rhoit I'm used to see this import instead:

```
from gi.repository import GObject
```

Is there any advantage on importing `_gobject`?

**Comment by jcollado** [comment]
Tested in Ubuntu 15.04 and worked fine.

Once the comment above is addressed the code can be merged. There are a few `pygtk` text strings in the demo and in the UI itself that should be updated to `Gtk`, but those can be updated after the merge.

File: browser.py (hunk 1 above)
Comment (rhoit) [inline browser.py]: @jcollado I got no Idea, ;) I'm still learing Gtk3.
I was looking closest possible match for `gobject` module from `gi` and never even looked into `gi.repository` coz `_gobject` was not giving any trouble.

Comment (rhoit) [inline browser.py]: I just updated to GObject, it work ok! :)

**Comment by jcollado** [comment]
@rhoit Merging now. Thanks!

**Pull request merged**